Resources/Prototypes/_Sunrise/Entities/Objects/Specific/Robotics/borg_tools.yml

```
@@ -267,6 +269,7 @@
     solution: hypospray
   - type: Hypospray
     onlyAffectsMobs: true
+    doAfterTime: 0.1 #Sunrise-Edit
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)